pkg/cmds/config/capa.go

```diff
@@ -29,8 +29,91 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	awsManagedControlPlane     = "AWSManagedControlPlane"
+	awsManagedMachinePool      = "AWSManagedMachinePool"
+	machinePool                = "MachinePool"
+	cluster                    = "Cluster"
+	controlplaneRoleAnnotation = "eks.amazonaws.com/controlplane-role"
+)
+
+func managedCPCIDR(ri *parser.ResourceInfo, vpcCidr string) error {
+	netcfg := map[string]any{
+		"vpc": map[string]any{
+			"cidrBlock": vpcCidr,
+		},
+	}
+	if err := unstructured.SetNestedMap(ri.Object.UnstructuredContent(), netcfg, "spec", "network"); err != nil {
+		return err
+	}
+	return nil
+}
+
+func managedCPRole(ri *parser.ResourceInfo, roleName string) error {
+	if err := unstructured.SetNestedField(ri.Object.UnstructuredContent(), roleName, "spec", "roleName"); err != nil {
+		return err
+	}
+	return nil
+}
+
+func managedMPScaling(ri *parser.ResourceInfo, minNodeCount, maxNodeCount int64) error {
+	scaling := map[string]any{
+		"minSize": minNodeCount,
+		"maxSize": maxNodeCount,
+	}
+	if err := unstructured.SetNestedMap(ri.Object.UnstructuredContent(), scaling, "spec", "scaling"); err != nil {
+		return err
+	}
+	return nil
+}
+
+func managedMPRole(ri *parser.ResourceInfo, roleName string) error {
+	if err := unstructured.SetNestedField(ri.Object.UnstructuredContent(), roleName, "spec", "roleName"); err != nil {
+		return err
+	}
+	return nil
+}
+
+func clusterAnnotations(ri *parser.ResourceInfo, managedControlplaneRole string) error {
+	if err := unstructured.SetNestedField(ri.Object.UnstructuredContent(), managedControlplaneRole, "metadata", "annotations", controlplaneRoleAnnotation); err != nil {
+		return err
+	}
+	return nil
+}
+
```

Comment:
Add SetAWS before function name to keep things similar.  
managedMPScaling() -> SetAWSManagedMPScaling()